frontend/src/__tests__/cypress/cypress/tests/mocked/projects/tabs/clusterStorage.cy.ts

```diff
@@ -226,7 +226,11 @@ describe('ClusterStorage', () => {
 
     cy.interceptK8sList(PVCModel, mockK8sResourceList([mockPVCK8sResource({})]));
 
-    addClusterStorageModal.findSubmitButton().click();
+    // verify text here:
+    const submitBtn = addClusterStorageModal.findSubmitButton();
+    submitBtn.should('contain.text', 'Add storage');
```

Comment:
testing text content can be fragile. its better to test functionality rather than the text is expected.... i.e does it create vs update